src/languages/c-like.js

```diff
@@ -18,6 +18,9 @@ export default function(hljs) {
   function optional(s) {
     return '(?:' + s + ')?';
   }
+  var C_LINE_COMMENT_MODE = hljs.COMMENT('//', '$', {
+    contains: [{begin: /\\\n/}]
```

Comment:
Should this be `\\$` since we're always in multi-line mode?

Reply:
`$` doesn’t work here (presumably since it’s a zero-width assertion, and the newline isn’t consumed?)

Reply:
Duh. :-)

Reply:
In fact, `\\(?=$).` would work — but only in single-line mode, since `.` otherwise fails to match a newline.